src/rnp/fficli.cpp

```diff
@@ -69,6 +69,8 @@
 #include <regex>
 #endif
 
+#define SHA1_KEY_FROM 1705629600
```

Comment:
More correct would be to obtain this via the `rnp_get_security_rule()` call, see tests for examples.